packages/api/src/generated/graphql.tsx

```diff
@@ -313,7 +313,7 @@ export type Taak = {
 
 export type TaakFormulier = {
   __typename?: 'TaakFormulier';
-  type: Scalars['String'];
+  formuliertype: Scalars['String'];
```

Comment:
I still think it's a bit of an ugly solution, but okay, as this seemed like the simplest solution.